pkg/newreleases/newreleases.go

```diff
@@ -246,19 +246,65 @@ func (nr NewReleases) ApplyLocalConfig(options ApplyLocalConfigOptions) error {
 	}
 
 	for _, projectCfg := range diff.MissingOnRemote {
+		emailNotification, err := emailNotificationFromString(projectCfg.EmailNotification)
+		if err != nil {
+			return err
+		}
 		projectOptions := newreleases.ProjectOptions{
-			EmailNotification:  (*newreleases.EmailNotification)(&projectCfg.EmailNotification),
+			EmailNotification:  &emailNotification,
 			WebhookIDs:         []string{},
 			Exclusions:         ExclusionSliceFromCfg(projectCfg.Exclusions),
 			ExcludePrereleases: &projectCfg.ExcludePrereleases,
 			ExcludeUpdated:     &projectCfg.ExcludeUpdated,
 			Note:               util.Ref(""),
 			TagIDs:             []string{},
 		}
-		_, err := nr.client.Projects.Add(context.Background(), projectCfg.Provider, projectCfg.Name, &projectOptions)
+		_, err = nr.client.Projects.Add(context.Background(), projectCfg.Provider, projectCfg.Name, &projectOptions)
 		if err != nil {
 			return err
 		}
 	}
 	return nil
 }
+
+func emailNotificationFromString(notification string) (newreleases.EmailNotification, error) {
+	var emailNotification newreleases.EmailNotification
+	switch notification {
+	case "none":
+		emailNotification = newreleases.EmailNotificationNone
+	case "instant":
+		emailNotification = newreleases.EmailNotificationInstant
+	case "hourly":
+		emailNotification = newreleases.EmailNotificationHourly
+	case "daily":
+		emailNotification = newreleases.EmailNotificationDaily
+	case "weekly":
+		emailNotification = newreleases.EmailNotificationWeekly
+	case "default":
+		emailNotification = newreleases.EmailNotificationDefault
+	default:
+		return "", fmt.Errorf("failed to parse email configuration: %q is not valid value", notification)
+	}
+	return emailNotification, nil
+}
+
+func emailNotificationToString(notification newreleases.EmailNotification) string {
+	var s string
+	switch notification {
+	case newreleases.EmailNotificationNone:
+		s = "none"
+	case newreleases.EmailNotificationInstant:
+		s = "instant"
+	case newreleases.EmailNotificationHourly:
+		s = "hourly"
+	case newreleases.EmailNotificationDaily:
+		s = "daily"
+	case newreleases.EmailNotificationWeekly:
+		s = "weekly"
+	case newreleases.EmailNotificationDefault:
+		s = "default"
+	default:
+		s = "none"
+	}
+	return s
```

Comment:
Suggest to try upstream these changes to their repo <https://github.com/newreleasesio/client-go>, to add `.String() string` and a `ParseEmailNotification(string) (EmailNotification, error)` function there. No clue if they would accept it, but could be a nice thing to do.